cocos/ui/editbox/edit-box.ts

```diff
@@ -478,10 +478,13 @@ export class EditBox extends Component {
 
     /**
      * @deprecated since v3.5.0, this is an engine private interface that will be removed in the future.
+     * @param text content filtered by sensitive words.This parameter may be undefined.
```

Comment:
```suggestion
     * @param text content filtered by sensitive words.This parameter may be undefined.
```
there is a strange character

Reply:
Already fixed